var/spack/repos/builtin/packages/py-nltk/package.py

```diff
@@ -41,14 +47,16 @@ class PyNltk(PythonPackage):
         destination="nltk_data/misc",
         placement="mwa_ppdb",
     )
+
     resource(
         name="punkt",
         url="https://raw.githubusercontent.com/nltk/nltk_data/gh-pages/packages/tokenizers/punkt.zip",
```

Comment:
Is this a rolling release file? Is it possible to pin it to a specific version so the checksum doesn't change?